happy-travel/edo
# Feature/aa 1508 add company currencies and new requisites

Issue: https://github.com/happy-travel/agent-app-project/issues/1508
Tested through swagger and insomnia

Test cases:
 1) Check if Voucher and Invoice generation work well
 2) Check admin endpoint for get and update CompanyInfo in static data where availableCurrencies field was added
